### ISSUE TITLE
staticDetectorGeo() is missing to allow for DBT in python

### ISSUE THREAD

**Issue by AnderBiguri** [CERN/TIGRE#280]

**Comment by yliu88au** [comment]
Will this function work?

[staticDetectorGeo.zip](https://github.com/CERN/TIGRE/files/7667433/staticDetectorGeo.zip)

**Comment by AnderBiguri** [comment]
@yliu88au  yes it should. I have no way of validating it, have you tried it with DBT data?

**Comment by yliu88au** [comment]
I have only tested it on a simulated case of tomosynthesis (construct an image, then projecting and reconstruction), it all makes sense. But it is still not the real data.

**Comment by AnderBiguri** [comment]
@yliu88au  if you simulated it outside TIGRE, then that is good enough! want to make a PR?
